src/applications/mhv/medications/containers/PrescriptionDetails.jsx

```diff
@@ -75,6 +75,8 @@ const PrescriptionDetails = () => {
       if (prescription) {
         focusElement(document.querySelector('h1'));
         updatePageTitle(`${prescription.prescriptionName} | Veterans Affairs`);
+      } else {
+        window.scrollTo(0, 0);
```

Comment:
@gerard-sullivan I think there is a platform utility for this under:
import scrollToTop from '@department-of-veterans-affairs/platform-utilities/ui/scrollToTop';

I would test to make sure its working, I only saw examples of the deprecated version being used